<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1011</width>
    <height>581</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1011</width>
    <height>581</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1011</width>
    <height>581</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(252, 237, 218);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>1091</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(252, 237, 218);
</string>
    </property>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>961</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget
{
 
	background-color: rgb(47, 60, 126);
   border-radius:20%;
}</string>
     </property>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>90</y>
        <width>75</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(2,0,36,1), stop:0.3 rgba(9,9,121,1), stop:1 rgba(0,212,255,1));
	
	
	
	color: rgb(222, 184, 135);
	border-radius:10%;
}

QPushButton:hover{
	
	color: rgb(255, 255, 255);
}background-color:rgb(37,150,190);
	color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>START</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>90</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(2,0,36,1), stop:0.3 rgba(9,9,121,1), stop:1 rgba(0,212,255,1));
	
	
	
	color: rgb(222, 184, 135);
	border-radius:10%;
}

QPushButton:hover{
	
	color: rgb(255, 255, 255);
}background-color:rgb(37,150,190);
	color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>PAUSE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>90</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(2,0,36,1), stop:0.3 rgba(9,9,121,1), stop:1 rgba(0,212,255,1));
	
	
	
	color: rgb(222, 184, 135);
	border-radius:10%;
}

QPushButton:hover{
	
	color: rgb(255, 255, 255);
}background-color:rgb(37,150,190);
	color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>RESUME</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>30</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(222, 184, 135);</string>
      </property>
      <property name="text">
       <string>SCRAPPING</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>30</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(222, 184, 135);</string>
      </property>
      <property name="text">
       <string>Progress</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>70</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#deb887;&quot;&gt;Time&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>30</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(222, 184, 135);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>90</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(2,0,36,1), stop:0.3 rgba(9,9,121,1), stop:1 rgba(0,212,255,1));
	
	
	
	
	color: rgb(222, 184, 135);
	border-radius:10%;
}

QPushButton:hover{
	
	color: rgb(255, 255, 255);
}background-color:rgb(37,150,190);
	color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>STOP</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>171</width>
       <height>321</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget
{
 
	background-color: rgb(47, 60, 126);
   border-radius:20%;
}</string>
     </property>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
 background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgb(222, 184, 135), stop:1 rgb(240, 53, 218));
	
	
	
	color: rgb(0, 0, 0);
	border-radius:10%;
}

QPushButton:hover{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgb(61, 217, 245), stop:1 rgb(240, 53, 218));
	color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string>Scraping</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>101</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
 background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgb(222, 184, 135), stop:1 rgb(240, 53, 218));
	
	
	
	color: rgb(0, 0, 0);
	border-radius:10%;
}

QPushButton:hover{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgb(61, 217, 245), stop:1 rgb(240, 53, 218));
	color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string>Sorting</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>170</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
 background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgb(222, 184, 135), stop:1 rgb(240, 53, 218));
	
	
	
	color: rgb(0, 0, 0);
	border-radius:10%;
}

QPushButton:hover{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgb(61, 217, 245), stop:1 rgb(240, 53, 218));
	color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string>Searching</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>230</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(2,0,36,1), stop:0.3 rgba(9,9,121,1), stop:1 rgba(0,212,255,1));
	
	
	
	color: rgb(222, 184, 135);
	border-radius:10%;
}

QPushButton:hover{
	
	color: rgb(255, 255, 255);
}background-color:rgb(37,150,190);
	color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>160</y>
       <width>591</width>
       <height>371</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>author</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>price</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>words</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>description</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>date</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>category</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>langguage</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
